functions/tsconfig.json

```diff
@@ -8,7 +8,8 @@
     "removeComments": true,
     "sourceMap": true,
     "strict": true,
-    "target": "es2017"
+    "target": "es2017",
+    "types": ["jest"]
```

Comment:
Saw an odd error when parsing the types in `@types/superagent` in the parent's (i.e. CLUE's) `node_modules` folder. 🤯  It turns out that by default the TypeScript compiler processes _all_ `@types/*` modules in _all_ accessible _node_modules_ folders. 😱  This prevents that.